package.json

```diff
@@ -21,7 +21,8 @@
     "Antonio Santiago Pérez <asantiago@cartodb.com>",
     "Ivan Malagon <ivan@carto.com>",
     "Ruben Moya <ruben@carto.com>",
-    "Carlos Matallín <matallo@carto.com>"
+    "Carlos Matallín <matallo@carto.com>",
+    "Jesús Arroyo Torrens <jarroyo@cartodb.com>"
   ],
   "license": "BSD-3-Clause",
   "dependencies": {
```

Comment:
IMO lint things should be part of the dev-deps

Reply:
What do you mean? eslint and grunt-eslint are in devDependencies.

Reply:
ouch you are right. sorry I was looking at the diff collapsed